app/src/androidTest/java/network/loki/messenger/HomeActivityTests.kt

```diff
@@ -72,25 +79,34 @@ class HomeActivityTests {
         onView(isRoot()).perform(waitFor(500))
     }
 
+    private fun objectFromDesc(id: Int) = device.findObject(By.desc(context.getString(id)))
+
     private fun setupLoggedInState(hasViewedSeed: Boolean = false) {
         // landing activity
-        onView(withId(R.id.registerButton)).perform(ViewActions.click())
-        // session ID - register activity
-        onView(withId(R.id.registerButton)).perform(ViewActions.click())
+        objectFromDesc(R.string.onboardingAccountCreate).click()
+
         // display name selection
-        onView(withId(R.id.displayNameEditText)).perform(ViewActions.typeText("test-user123"))
-        onView(withId(R.id.registerButton)).perform(ViewActions.click())
+        objectFromDesc(R.string.displayNameEnter).click()
+        device.pressKeyCode(65)
```

Comment:
Maybe add a comment like "Enter the dummy name 'ABC'" (is it sad that I remember some ASCII codes? Yes it is, haha)

Reply:
This is code from the onboarding PR. I suggest looking at the commits instead to see the work from this PR.